src/components/v5/common/Fields/InputBase/FormattedInput.tsx

```diff
@@ -96,7 +101,10 @@ const FormattedInput: FC<FormattedInputProps> = ({
            */
           options={options}
           onInit={(cleaveInstance) => setCleave(cleaveInstance)}
-          onChange={onChange}
+          onChange={(e) => {
+            handlePrefixTrim(options.prefix);
```

Comment:
Name of this function is quite misleading as this does not trim the prefix but it's moving the cursor to correct position

Reply:
As I understand it we're no longer doing prefix/suffix token symbol for the staking input. 

@arrenv please confirm if that's still the case